app/models/data/returning_users.rb

```diff
@@ -19,44 +16,33 @@ def dashboard
           {
             weekly: weekly,
             monthly: monthly,
-            bimonthly: bimonthly,
             quarterly: quarterly,
           },
         ]
       end
 
     private
 
-      # @param previous [Range]
-      # @param current [Range]
+      # @Param [Range<Time>] time_range
```

Comment:
`@param` ? does yard complain if you compile it?

Reply:
Yes [f38d173](https://github.com/DFE-Digital/early-years-foundation-recovery/pull/749/commits/f38d1738264e7d3996999db663972aef998d1e54)